src/style.css

```diff
@@ -1,3 +1,78 @@
+@font-face {
+  font-family: Roboto;
+  src: url(fonts/Roboto-Medium.ttf);
+  font-weight: 500;
+}
+
+:root {
+  --color-ref: #00acdc;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  background-color: white;
+}
+
+.header {
+  display: flex;
+  width: auto;
+  justify-content: space-between;
+  padding: 0 50px;
+  align-items: center;
+}
+
+.logoImagem {
+  height: 40px;
+}
+
+.navigation {
+  height: 60px;
+}
+
+.navigation .listUn {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+  display: flex;
+  height: 100%;
+  align-items: center;
+}
+
+.navigation .listLinks:not(:last-child) {
+  margin-right: 20px;
+}
+
+.navigation .listLinks:first-child {
+  color: var(--color-ref);
+}
+
+.navigation a {
```

Comment:
Instead of using specification here, you could add the styling to a class unique to the link. Would be a better approach